packages/asc-core/src/components/Radio/RadioLabel.ts

```diff
@@ -1,24 +1,32 @@
 import styled from '../../styled-components'
+import getThemeColor from '../../utils/getThemeColor'
 
-interface Props {
+type Props = {
   htmlFor: string
   disabled?: boolean
 }
 
-const Label = styled.label.attrs<Props>({
-  htmlFor: ({ htmlFor }: Props) => htmlFor,
-  disabled: ({ disabled }: Props) => disabled,
-})<Props>`
-  color: ${props => (props.disabled ? '#bebebe' : '#434343')};
+const Label = styled.label.attrs<Props>(props => ({
+  htmlFor: props.htmlFor,
+  disabled: props.disabled,
+}))<Props>`
+  color: ${({ theme, disabled }) =>
```

Comment:
Shouldn't the values, that are used for styling (like `16px`, `28px`, `4px`, `AvenirNextLTW01-Regular` and such) come from the theme configuration?